src/sidebar/components/group-list.js

```diff
@@ -12,6 +14,18 @@ function GroupListController($window, analytics, groups, settings, serviceUrl) {
     $window.open(serviceUrl('groups.new'), '_blank');
   };
 
+  this.focusedIcon = function() {
+    const focusedGroup = this.groups.focused();
+    return focusedGroup && (
+      focusedGroup.organization.logo || this.thirdPartyGroupIcon
+    );
+  };
+
+  this.focusedIconClass = function() {
+    const focusedGroup = this.groups.focused();
+    return (focusedGroup && focusedGroup.type === 'open') ? 'public' : 'group';
```

Comment:
This means that for restricted groups we will be showing the icon that we've previously used for private groups. Was that intended?

Reply:
Not intentional! Possible mistake. Thus: restricted groups should show world/public icon (fallback)? I'll update it.

Reply:
World/public icon for restricted groups makes sense to me, since they are world-readable.